doc-content/jbpm-docs/src/main/asciidoc/GettingStarted-business-proc.adoc

```diff
@@ -19,9 +19,9 @@ You will then refer to the *Mortgage_Process* sample project included in {CENTRA
 
 include::GettingStartedBusProc/intro7-con.adoc[leveloffset=+1]
 
-include::../../../../{drools-dir}/Examples/decision-examples-central-con.adoc[leveloffset=+1]
+//include::../../../../{drools-dir}/Examples/decision-examples-central-con.adoc[leveloffset=+1]
 
-include::../../../../{drools-dir}/Examples/decision-examples-central-proc.adoc[leveloffset=+2]
+//include::../../../../{drools-dir}/Examples/decision-examples-central-proc.adoc[leveloffset=+2]
```

Comment:
I would leave examples in jbpm as otherwise there is no such list of example projects in the jbpm docs. These examples are not only for decisions and it is good to list them I think.

Reply:
Thank you for your input @marianmacik I have already pushed the changes upstream, but am adding a note about this comment to the next work item: https://issues.redhat.com/browse/BXMSDOC-6883